### PR TITLE
This is Prison v3.2.0

### PR DESCRIPTION
**Pull request by Perksey**

### CODE REVIEW

File: prison-core/src/main/java/tech/mcprison/prison/Prison.java [hunk 1]
```diff
@@ -50,6 +50,8 @@
  */
 public class Prison implements PluginEntity {
 
+	public static final int SPIGOTMC_ORG_PROJECT_ID = 72740;
```
Comment (Perksey) [inline prison-core/src/main/java/tech/mcprison/prison/Prison.java]: Pleaase change this to the official resource.

File: prison-core/src/main/resources/lang/mines/de_DE.properties [hunk 2]
```diff
@@ -27,3 +27,4 @@ block_not_removed=&7Dieser Block ist nicht in dieser Mine vorhanden.
 block_deleted=&7Block &3%1 &7von der Mine entfernt &3%2&7.
 mine_redefined=&7Erfolgreich die Mine &3neudefiniert &7.
 missing_world=&7Die Welt, in der die Mine erstellt wurde, kann nicht gefunden werden.
+block_search_blank=&7Enter a value to search for a block.&7.
```
Comment (Perksey) [inline prison-core/src/main/resources/lang/mines/de_DE.properties]: Remove this, the localization system defaults to the english variant anyway.

File: prison-core/src/main/resources/lang/mines/en_US.properties [hunk 3]
```diff
@@ -27,3 +27,4 @@ block_not_removed=&7That block is not in this mine.
 block_deleted=&7Removed block &3%1 &7from mine &3%2&7.
 mine_redefined=&7Successfully &3redefined &7the mine.
 missing_world=&7The world that the mine was created in can't be found.
+block_search_blank=&7Enter a value to search for a block.&7.
```
Comment (Perksey) [inline prison-core/src/main/resources/lang/mines/en_US.properties]: Remove this, the localization system defaults to the english variant anyway.

File: prison-core/src/main/resources/lang/mines/es_ES.properties [hunk 4]
```diff
@@ -27,3 +27,4 @@ block_not_removed=&7Ese bloque no se encuentra en esta mina.
 block_deleted=&7Se ha eliminado el bloque &3%1 &7de la mina &3%2&7.
 mine_redefined=&7Se ha &3redefinido &7la mina exitosamente.
 missing_world=&7El mundo en el que se ha creado la mina no se ha podido encontrar.
+block_search_blank=&7Enter a value to search for a block.&7.
```
Comment (Perksey) [inline prison-core/src/main/resources/lang/mines/es_ES.properties]: Remove this, the localization system defaults to the english variant anyway.

File: prison-core/src/main/resources/lang/mines/hu_HU.properties [hunk 5]
```diff
@@ -27,3 +27,4 @@ block_not_removed=&7Ez a blokk nem ebben a bányában van.
 block_deleted=&7A blokk törölve &3%1 &7a(z) &3%2&7 bányából.
 mine_redefined=&7A bányát sikeresen &3újradefiniálta&7.
 missing_world=&7Nem található meg a világ, amelyet a bányát hoztak létre.
+block_search_blank=&7Enter a value to search for a block.&7.
```
Comment (Perksey) [inline prison-core/src/main/resources/lang/mines/hu_HU.properties]: Remove this, the localization system defaults to the english variant anyway.

File: prison-core/src/main/resources/lang/mines/it_IT.properties [hunk 6]
```diff
@@ -26,4 +26,5 @@ block_set=&7Cambiato il blocco &3%1 &7nella miniera &3%2&7.
 block_not_removed=&7Quel blocco non è in quella miniera.
 block_deleted=&7Rimosso il blocco &3%1 &7dalla miniera &3%2&7.
 mine_redefined=&7E' con successo stata &3ridefinita &7la miniera.
-missing_world=&7Il mondo nel quale è stata creata la miniera non è stato trovato.
\ No newline at end of file
+missing_world=&7Il mondo nel quale è stata creata la miniera non è stato trovato.
+block_search_blank=&7Enter a value to search for a block.&7.
```
Comment (Perksey) [inline prison-core/src/main/resources/lang/mines/it_IT.properties]: Remove this, the localization system defaults to the english variant anyway.

File: prison-core/src/main/resources/lang/mines/nl_BE.properties [hunk 7]
```diff
@@ -24,4 +24,5 @@ block_set=&7Blok &3%1 &7veranderd &7in mijn &3%2&7.
 block_not_removed=&7Die blok kon niet verwijderd worden.
 block_deleted=&7Blok &3%1 verwijderd &7van mijn &3%2&7.
 mine_redefined=&7Succesvol de mijn &3geherdefineerd.
-missing_world=&7De wereld van deze mijn kon niet worden gevonden.
\ No newline at end of file
+missing_world=&7De wereld van deze mijn kon niet worden gevonden.
+block_search_blank=&7Enter a value to search for a block.&7.
```
Comment (Perksey) [inline prison-core/src/main/resources/lang/mines/nl_BE.properties]: Remove this, the localization system defaults to the english variant anyway.

File: prison-core/src/main/resources/lang/mines/nl_NL.properties [hunk 8]
```diff
@@ -24,4 +24,5 @@ block_set=&7Blok &3%1 &7veranderd &7in mijn &3%2&7.
 block_not_removed=&7Die blok kon niet verwijderd worden.
 block_deleted=&7Blok &3%1 verwijderd &7van mijn &3%2&7.
 mine_redefined=&7Succesvol de mijn &3geherdefineerd.
-missing_world=&7De wereld van deze mijn kon niet worden gevonden.
\ No newline at end of file
+missing_world=&7De wereld van deze mijn kon niet worden gevonden.
+block_search_blank=&7Enter a value to search for a block.&7.
```
Comment (Perksey) [inline prison-core/src/main/resources/lang/mines/nl_NL.properties]: Remove this, the localization system defaults to the english variant anyway.

**Comment by Perksey** [comment]
@rbluer

File: prison-core/src/main/java/tech/mcprison/prison/Prison.java (hunk 1 above)
Comment (rbluer) [reply prison-core/src/main/java/tech/mcprison/prison/Prison.java]: already did.  That's in the commits that still need to be pulled.

File: prison-core/src/main/resources/lang/mines/de_DE.properties (hunk 2 above)
Comment (rbluer) [reply prison-core/src/main/resources/lang/mines/de_DE.properties]: Just fixed these and pushed the commit to bleeding, so should be there by now.

**Comment by rbluer** [comment]
I am not seeing the final commits from my old fork.  Without those most recent commits, this plugin will not work.

File: changelog_v3.2.x.md [hunk 9]
```diff
@@ -0,0 +1,533 @@
...
+Enhanced the /mines list to show more details about the mines, plus provide a clickable way to TP to each mine.
+
+
+**Build artifacts:**  
+  Be aware that this build artifact is not production grade and still needs more testing.
+  * **prison_v3.5.2-beta.1.jar**    
+
+
+
+## tag v3.2.0-alpha.6 - 2019-11-24 (v3.5.2-alpha.6)
```
Comment (Perksey) [inline changelog_v3.2.x.md]: Final nitpick while I consult with Faizaan on how we're going about coordinating this release: can you coalesce these unofficial tags into one big set of release notes for 3.2.0?

Comment (rbluer) [reply changelog_v3.2.x.md]: FYI the merged and cleaned up release notes for v3.2.0 is in commit b6df91e in bleeding.

**Pull request merged**